README.md

```diff
@@ -1,23 +1,59 @@
-# govwifi-admin
+# Govwifi Admin API
 
-## Purpose
+This is the admin platform, a website where organisations can create and manage their GovWifi installation within their organisation.
+The private GovWifi [build repository][build-repo] contains instructions on how to build GovWifi end-to-end - the sites, services and infrastructure.
 
-This is the User Admin UI for the [GovWiFi][link_govwifi] project.
+## Overview
 
-## How to install and use
+The API contains files and test suites that describe and maintain the desired functionality of the admin platform. It is primarily a Ruby-on-Rails Project.
```

Comment:
This line doesn't really make much sense to me. Something better here would perhaps be a list of the features. For instance.

```
The application allows users to perform the following tasks:
- Create an admin account.
- Invite team members to their account.
- View instructions on how to setup and configure GovWifi on their local network.
- Add IP addresses of their access points to the GovWifi system.
- View logs of authentication requests to GovWifi by IP and username.
- Make support ticket requests.

The application also includes a "Super Admin" login that allows the GDS administrator to:
- View all organisations signed up to GovWifi
- See specific information on each of these organisations
- Add custom organisation names to the allowed register
```